samples/apps/logging/logging.cpp

```diff
@@ -164,7 +168,7 @@ namespace loggingapp
         .install();
 
       // SNIPPET_START: get
-      auto get = [this](ccf::ReadOnlyEndpointContext& args, nlohmann::json&&) {
+      auto get = [this](auto& args, nlohmann::json&&) {
```

Comment:
Nitpick: Some of these are called `args`, others `ctx`. Should we unify?

Reply:
That's a great post-1.0 task #2345